zbarlight/__init__.py

```diff
@@ -1,5 +1,24 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ._zbarlight import qr_code_scanner
+from ._zbarlight import zbar_code_scanner
+
+from PIL import Image
 
 __version__ = '0.1.1'
-__ALL__ = ['qr_code_scanner']
+__ALL__ = ['scan_codes', 'qr_code_scanner']
+
+
+def qr_code_scanner(image, width, height):
+    """Specific QR code scanner (deprecated)"""
+    result = zbar_code_scanner(b'qr.enable', image, width, height)
+    if isinstance(result, list) and len(result) != 1:
+        return None
+    return result[0] if result is not None else None
```

Comment:
I think that `zbar_code_scanner()` returns `None` or a list. We could hence write:

    if isinstance(result, list) and len(result) == 1:
        return result[0]
    return None

... which looks simpler.